website/content/v1.11/kubernetes-guides/network/deploying-calico.md

```diff
@@ -0,0 +1,195 @@
+---
+title: "Deploying Calico CNI"
+description: "In this guide you will learn how to set up Calico CNI on Talos in two mode eBPF and NFtables."
+---
+
+This documentation is designed to get you up and running with Talos and Calico CNI. Since both Calico and Talos support multiple networking technologies, you will learn how to run your environment with both the [Calico eBPF dataplane](https://docs.tigera.io/calico/latest/operations/ebpf/enabling-ebpf) and [NFTables](https://docs.tigera.io/calico/latest/getting-started/kubernetes/nftables). Optionally, you can also enable Calico's network [observability stack](https://docs.tigera.io/calico/latest/observability/) to gain insights into your cluster networking and policy behavior.
+
+## Configuring Talos
+
+To install Calico, you first need to disable the default CNI. This can be done by applying a patch file during cluster creation.
+The store the following YAML template in a file (`patch.yaml`).
+
+```yaml
+cluster:
+  network:
+    cni:
+      name: none
+```
+
+After generating the patch file add the `--config-patch` argument to your `talosctl gen config`.
+
+```bash
+talosctl gen config \
+    my-cluster https://calico-talos.local:6443 \
+    --config-patch @patch.yaml
+```
+
+## Installing Tigera Operator
+
+Recommended way to install Calico is via `Tigera-operator` manifest. The operator will make sure that all Calico components are always up and running.
+
+> **Note** If you like to install Calico using Helm [checkout this document](https://docs.tigera.io/calico/latest/getting-started/kubernetes/helm).
+
+Use the following command to install the latest Tigera operator.
+
+```bash
+kubectl create -f https://docs.tigera.io/calico/latest/manifests/tigera-operator.yaml
+```
+
+### Configuring Calico Networking
+
+Calico has a pluggable dataplane architecture that lets you choose the networking technology based on your use case. You can configure the dataplane by setting the `linuxDataplane` key in the installation manifest.
+
+> **Note** If you like to learn more about the available Calico configurations [checkout this document](https://docs.tigera.io/calico/latest/reference/installation/api).
+
+{{< tabpane text=true >}}
+{{% tab header="eBPF" %}}
+
+By default, Calico uses the `/var` directory to mount cgroups. However, since this path is not writable in Talos, you need to change it to `/sys/fs/cgroup`.
+
+Use the following command to update the cgroup mount path:
+
+```bash
+kubectl create -f -<<EOF
+apiVersion: crd.projectcalico.org/v1
+kind: FelixConfiguration
+metadata:
+  name: default
+spec:
+  cgroupV2Path: "/sys/fs/cgroup"
```

Comment:
FYI this was added in https://github.com/projectcalico/calico/commit/984ab6e which is *not* included in the current release of v3.30.3.